rollup/circuitcapacitychecker/libzkp/Makefile

```diff
@@ -6,5 +6,5 @@ clean:
 
 libzkp:
 	cargo build --release
-	cp $(PWD)/target/release/libzkp.so $(PWD)/
-	find $(PWD)/target | grep libzktrie.so | xargs -I{} cp {} $(PWD)/
+	sudo cp $(PWD)/target/release/libzkp.so /usr/local/lib/
+	find $(PWD)/target | grep libzktrie.so | xargs -I{} sudo cp {} /usr/local/lib/
```

Comment:
don't move to `/usr/local/lib/`, because we may not have the privilege.

just add `$(PWD)/rollup/circuitcapacitychecker/libzkp` to `LD_LIBRARY_PATH`